docs/proposals/custom-playbook-names.md

```diff
@@ -0,0 +1,58 @@
+# Custom Playbook Names
+
+## Introduction
+Instead of forcing the user to have a playbook named provision, deprovision,
+bind, or unbind, allow the user to create a single playbook and select the
+broker action at the role or task level.
+
+
+## Problem Description
+If I want to convert my ansible playbooks to an APB, it's required that I create
+provision, bind, unbind, or deprovision playbook(s). This is inconvenient
+because I would need to change my playbook name, documentation, possibly some
+variables, and break any existing users just so I can support the APB format.
+
+If I want to have multiple APBs in a single git repo I get a similar issue.
```

Comment:
It would help me to see a more concrete example, maybe even with the relevant portion of the file tree, when discussing this. It is not immediately obvious to my why we need to modify the apb-base to handle this. Some changes to your APBs dockerfile could accomplish this for you.

Reply:
+1 A concrete example would be helpful.

Reply:
Here's an example file tree: https://github.com/openshift/ansible-service-broker/pull/815/commits/d219b713b64b8c59fe293d892b70733f1ff7f16b